templates/includes/navbar.html

```diff
@@ -18,6 +18,10 @@
                     </a>
                     <div class="dropdown-menu" aria-labelledby="navbarHackathons">
                         <a class="dropdown-item" href="{% url 'hackathon:hackathon-list' %}">Events</a>

```

Comment:
In line with my earlier comment, I would rename "Events" to "View Hackathons" and "Create Event" to "Create Hackathon" please.

Reply:
Done. The "Events" link wasn't one that I added by the way, but I've changed it while in the file.